kafka/dockerbuild.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+COMMIT_HASH=$(git rev-parse --short HEAD)
```

Comment:
Requires you to be in that folder. How about 

```
cd $( cd "$( dirname "$0" )"; pwd )
```

to change to the script's directory.

Reply:
I was just thinking about remove the whole file actually. If I got this right then there is a build trigger on that repo that builds the docker image with considering the `dockerbuild.sh`.